src/misc/ErrorHandler.ts

```diff
@@ -40,7 +40,11 @@ function produceThrottledFunction<R>(ms: number, fn: () => Promise<R>): () => Pr
  * */
 const importErrorHandler = produceThrottledFunction(200, () => import("./ErrorHandlerImpl.js"))
 
-export async function handleUncaughtError(e: Error) {
+export async function handleUncaughtError(e?: Error) {
+	if (e == null) {
```

Comment:
if we get here and the error is not there for some reason I think we should create a new error and act on it, we can at least get some stack trace, this should not happen normally

Reply:
isn't the uncaught error handler invoked from the top level after the original stack has been completely unwound?

Reply:
It can be:
 - `window` `error` event
 - `window` `unhandledrejection` event
 - the one from worker
 - the one from `NavButton`
 - the one from `ServiceWorkerClient`

we could at least guess which one it is that is misbehaving

Reply:
in the report where I can check it it's the unhandledRejection event with an undefined reason. It's also happening exclusively on safari.

Reply:
ah, okay then, should we at least print something? not nice if we completely eat up the errors

Reply:
yes, I can see that.